pkgs/misc/tmux-plugins/default.nix

```diff
@@ -818,4 +818,22 @@ in rec {
       sha256 = "hRvkBf+YrWycecnDixAsD4CAHg3KsioomfJ/nLl5Zgs=";
     };
   };
+
+  tmux-nova = mkTmuxPlugin rec {
+    pluginName = "tmux-nova";
+    version = "1.2.0";
+    src = fetchFromGitHub {
+      owner = "o0th";
+      repo = "tmux-nova";
+      rev = "v${version}";
+      sha256 = "16llz3nlyw88lyd8mmj27i0ncyhpfjj5c1yikngf7nxcqsbjmcnh";
+    };
+    meta = with lib; {
+      homepage = "https://github.com/o0th/tmux-nova";
+      description = "tmux-nova theme";
+      license = licenses.mit;
+      platforms = platforms.unix;
+      maintainers = with maintainers; [ o0th ];
```

Comment:
Please add yourself to the [maintainers](https://github.com/o0th/nixpkgs/blob/add-tmux-nova/maintainers/maintainer-list.nix) file? Check out the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md)

```
       error: undefined variable 'o0th'

          835|       platforms = platforms.unix;
          836|       maintainers = with maintainers; [ o0th ];
             |                                         ^
          837|     };
```